packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -0,0 +1,187 @@
+  transition: background-color var(--iui-duration-1) ease-out;
+  @media (forced-colors: active) {
+    --_iui-expandable-block-expander-icon-fill: CanvasText;
+  }
+
+  &:where(:hover) {
+    --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon-hover);
+    --_iui-expandable-block-header-caption-color: var(--iui-color-text-hover);
+    --_iui-expandable-block-header-label-color: var(--iui-color-text-hover);
+
+    background-color: var(--_iui-expandable-block-header-background-color-hover);
+
+    @media (forced-colors: active) {
+      --_iui-expandable-block-header-caption-color: Highlight;
+      --_iui-expandable-block-header-label-color: Highlight;
+      --_iui-expandable-block-expander-icon-fill: Highlight;
+      border-color: Highlight;
+    }
+  }
+
+  &[disabled],
+  &:disabled,
+  &[aria-disabled='true'] {
+    --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon-disabled);
+    --_iui-expandable-block-header-caption-color: var(--iui-color-text-disabled);
+    --_iui-expandable-block-header-label-color: var(--iui-color-text-disabled);
+
+    cursor: not-allowed;
+    background-color: var(--_iui-expandable-block-header-background-color-disabled);
+    border-color: var(--iui-color-border-disabled);
+
+    @media (forced-colors: active) {
+      --_iui-expandable-block-expander-icon-fill: GrayText;
+      --_iui-expandable-block-header-caption-color: GrayText;
+      --_iui-expandable-block-header-label-color: GrayText;
+      border-color: GrayText;
+    }
+  }
+}
+
+/// @arg $type Can be one of: expander, status
+@mixin iui-expandable-block-header-icons($type) {
+  @include iui-icon-style('m');
+
+  @if $type == 'expander' {
+    width: var(--_iui-expandable-block-expander-icon-size);
+    height: var(--_iui-expandable-block-expander-icon-size);
+    fill: var(--_iui-expandable-block-expander-icon-fill);
+    transform: var(--_iui-expandable-block-expander-icon-transform, none);
+    transition: fill var(--iui-duration-1) ease-out;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: fill var(--iui-duration-1) ease-out, transform var(--iui-duration-1) ease-out;
+    }
+  }
+
+  @if $type == 'status' {
+    @media (forced-colors: active) {
+      fill: CanvasText;
+    }
+  }
+}
+
+@mixin iui-expandable-block-header-label {
+  display: flex;
+  flex-direction: column;
+  flex: auto;
+  min-width: 0;
+  font-size: var(--iui-font-size-2);
+  color: var(--_iui-expandable-block-header-label-color, inherit);
+}
+
+/// @arg $type Can be one of: label, sublabel
+@mixin iui-expandable-block-header-label-text($type) {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  transition: color var(--iui-duration-1) ease;
+  @media (forced-colors: active) {
+    transition: none;
+  }
+
+  @if $type == 'sublabel' {
+    font-size: var(--iui-font-size-0);
+    color: var(--_iui-expandable-block-header-caption-color, var(--iui-color-text-muted));
```

Comment:
`--_iui-expandable-block-header-caption-color` is not found. So the forced colors don't get applied. In the screenshot, the variable is greyed out and not blue.

![image](https://user-images.githubusercontent.com/45748283/203425278-28062360-76ff-4cdb-9c98-496308be34c9.png)

Reply:
I _think_ this is working as expected.  I am both introducing and setting the default value for that new variable in the same line.  This:
```scss
color: var(--my-new-variable, blue);
```
Is the same as this:
```scss
--my-new-variable: blue;
color: var(--my-new-variable);
```

When I turn on `forced-colors` mode, I do see the hover & disabled effects taking place (in order of default state, hovered state, disabled state):
<img width="689" alt="Screenshot 2022-11-23 at 12 28 11 PM" src="https://user-images.githubusercontent.com/849817/203611183-19d31f65-17b8-4e1f-b691-680be8d5cfc9.png">

I'll give you a call just to confirm I'm not misunderstanding.

Reply:
>  I am both introducing and setting the default value for that new variable in the same line

Ohh, wow, this is a cool trick, didn't know this before. In that case, I think the missing variable indication in the inspector makes sense. Thanks for explaining this.